pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -79,6 +91,16 @@ func Open(ctx context.Context, DataPath, DBPath string) (db *DB, err error) {
 		return nil, err
 	}
 
+	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bwusagetbl` (`size` INT(10), `daystartdate` INT(10), `dayenddate` INT(10));")
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = tx.Exec("CREATE INDEX IF NOT EXISTS idx_bwusagetbl_size ON bwusagetbl (size);")
```

Comment:
We may not need to create an index anymore. The original intention was to create an index on date so we can easily get ranges of entries between two dates